polytope/polytope.py

```diff
@@ -201,19 +201,40 @@ def __copy__(self):
         P.fulldim = self.fulldim
         return P
 
+    def contains(self, point, abs_tol=ABS_TOL):
+        """Return True if polytope contains points.
+
+        @type point: array_like, 1d array, or 2d array (of column vectors)
+
+        @rtype: bool, 1d array
+
+        See Also
+        ========
+        L{is_inside}
+        """
+        if not isinstance(point, np.ndarray):
```

Comment:
I was also thinking about this today, but my thought was to use [`np.asanyarray`](https://docs.scipy.org/doc/numpy/reference/generated/numpy.asanyarray.html#numpy.asanyarray) instead of `isinstance` to reduce the number of lines of code.

However, I think your suggestion is correct. The function should only work if the user passes something that is a duck type of  `array`. For example, passing a `list` would not work because it does not have the parameters `ndim` or `shape`.